test-engineering/contract-tests/volumes/client/scenarios_tiles_cache.yml

```
@@ -0,0 +1,264 @@
+# Note: The scenarios in this file will be executed using a 'CONTILE_TILES_TTL' value
+# greater than 0. As a result, requests made to Contile will be cached, impacting
+# subsequent scenario executions.
+
+scenarios:
+  - name: success_tiles_cache_with_same_form_and_region
```

I suggest renaming this scenario because all proxy parameters that are included in the cache audience key need to be identical in order to get cached tiles.

```suggestion
  - name: success_tiles_cached_for_identical_proxy_params
```

---

Change applied: [012ef0d](https://github.com/mozilla-services/contile/pull/431/commits/012ef0de6a94ea0bc261999418d20435e40c333a)